### PR TITLE
Stablize SchemaSelectionPanelTest

### PR DESCRIPTION
**Pull request by abrooksv**
* Also fix the detekt violations

## Testing
Used `schemaTemplateParametersBuiltAfterSelectionCustomerUploadedSchema`
Before:
Screen Shot 2021-06-16 at 4 38 41 PM

After:
Screen Shot 2021-06-16 at 4 53 26 PM

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/wizard/SchemaSelectionPanelTest.kt [hunk 1]
```diff
@@ -223,13 +190,53 @@ class SchemaSelectionPanelTest {
...
+
+        return selector
     }
 
-    private fun load() {
-        initMockResourceCache()
-        schemaSelectionPanel.awsConnection = connectionManager.settingsManager.connectionSettings()
-        schemaSelectionPanel.reload()
-        schemaSelectionPanel.schemasSelector.waitToLoad()
+    private companion object {
```
Comment (manodnyab) [inline jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/wizard/SchemaSelectionPanelTest.kt:212]: Why are these moved to a companion object?

**Review by manodnyab — APPROVED** [review]

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/wizard/SchemaSelectionPanelTest.kt (hunk 1 above)
Comment (abrooksv) [reply jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/wizard/SchemaSelectionPanelTest.kt:212]: 1. Naming convnetion is all uppercase is for static values so  they should be in companion
2. Constant vals should be in static fields, not per instance
